lib/styleguide.js

```diff
@@ -254,15 +254,6 @@ function filterFiles(files, filter) {
 }
 
 module.exports.generate = function(options) {
-  try {
-    if (options.includeDefaultStyles) {
```

Comment:
At this point `options.includeDefaultStyles` hasn't gone through `common.sanitizeOptions()` so the default `includeDefaultStyles` is `false`.